pennylane/fermi/fermionic.py

```diff
@@ -109,8 +117,66 @@ def __repr__(self):
         r"""Terminal representation of a FermiWord"""
         return str(self)
 
+    def __add__(self, other):
+        """Add a FermiSentence, FermiWord or constant to a FermiWord. Converts both
+        elements into FermiSentences, and uses the FermiSentence __add__
+        method"""
+
+        self_fs = FermiSentence({self: 1.0})
+
+        if isinstance(other, FermiSentence):
+            return self_fs + other
+
+        if isinstance(other, FermiWord):
+            other_fs = FermiSentence({other: 1.0})
+            return self_fs + other_fs
```

Comment:
A little more compact
```suggestion
            return self_fs + FermiSentence({other: 1.0})
```